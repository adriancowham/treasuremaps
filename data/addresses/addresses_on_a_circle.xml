<posts>
<item rdf:about="http://sacramento.craigslist.org/gms/2339978106.html"> 
<title><![CDATA[Garage Sale 1 (Vacaville) $75]]></title> 
<link>http://sacramento.craigslist.org/gms/2339978106.html</link> 
<description><![CDATA[Two families household items, furniture, home décor, clothing, games, dvd’s, dishes, sporting goods,  and on and on and on…..
<br> 
426 Sussex Cir cross of Beelard between Elmira and Marshall
<br> 
Saturday 4/23/2011 8AM
<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Vacaville -->Location: Vacaville
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T22:54:17-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339978106.html</dc:source> 
<dc:title><![CDATA[Garage Sale 1 (Vacaville) $75]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T22:54:17-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2339886308.html"> 
<title><![CDATA[HUGE moving sale!! Sat-Sun (4/23-4/24) (North Natomas)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339886308.html</link> 
<description><![CDATA[HUGE moving sale this weekend!! We are moving and can't take everything with us. 
<br> 
Everything from living room furniture, bed room sets, dining room set (china hutch, table, and chairs), cooking accessories, clothing, knick-nacks, lamps,  and much more
<br> 
Saturday and Sunday 8-4
<br> 

<br> 
161 Jarvis Cir, 95834<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=North Natomas -->Location: North Natomas
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T20:09:48-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339886308.html</dc:source> 
<dc:title><![CDATA[HUGE moving sale!! Sat-Sun (4/23-4/24) (North Natomas)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T20:09:48-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339852989.html"> 
<title><![CDATA[Galt Garage Sale- toys & clothes (788 Morgans Ranch Circle)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339852989.html</link> 
<description><![CDATA[LOTS of girls clothing sizes 7-14 1/2. Tees starting @ $.25. Toys, movies & books too. Nearly new basketball shoes- girls size 4. Women's clothing. 
<br> 
Friday April 22 & Sat April 23- 8am to 1pm. Morgans Ranch is off Pinto Trail, off Lake Canyon Ave & Twin Cities (near Raley's)<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=788 Morgans Ranch Circle -->Location: 788 Morgans Ranch Circle
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T19:41:03-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339852989.html</dc:source> 
<dc:title><![CDATA[Galt Garage Sale- toys & clothes (788 Morgans Ranch Circle)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T19:41:03-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339683015.html"> 
<title><![CDATA[Yard Sale 4/22 & 4/23   9:00-2:00 (Greenhaven/Pocket Area)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339683015.html</link> 
<description><![CDATA[Yard Sale Fri 4/22 & Sat 4/23  9:00-2:00 each day. PLEASE NO EARLY BIRDS!!!!! Some furniture, clothing (ladies & Big/Tall mens), movies (dvds & vcr tapes), music (cds), some tupperware, some jewelry, ladies shoes, PS2 games and lots of other stuff. Come see to appreciate. Located at 30 Parkshore Circle, Sacramento, 95831. Again - PLEASE NO EARLY BIRDS!!!! Thank You.<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Greenhaven/Pocket Area -->Location: Greenhaven/Pocket Area
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T17:40:19-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339683015.html</dc:source> 
<dc:title><![CDATA[Yard Sale 4/22 & 4/23   9:00-2:00 (Greenhaven/Pocket Area)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T17:40:19-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339246141.html"> 
<title><![CDATA[Garage Sale (441 Colonial Circle, Vacaville)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339246141.html</link> 
<description><![CDATA[Garage Sale Friday 4/22 and Saturday 4/23 7:30am
<br> 
Lot of good stuff from:
<br> 
clothes
<br> 
dishes
<br> 
movies
<br> 
toys
<br> 
kids tents
<br> 
kids playhouse
<br> 

<br> 
Come get some good stuff for cheap
<br> 
Thank you!<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=441 Colonial Circle, Vacaville -->Location: 441 Colonial Circle, Vacaville
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T13:43:10-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339246141.html</dc:source> 
<dc:title><![CDATA[Garage Sale (441 Colonial Circle, Vacaville)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T13:43:10-07:00</dcterms:issued> 
</item>
</posts>