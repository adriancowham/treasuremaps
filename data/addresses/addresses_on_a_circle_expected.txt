426 Sussex Cir
161 Jarvis Cir
788 Morgans Ranch Circle
30 Parkshore Circle
441 Colonial Circle